Feat/countries/index-page

npthang opened the pull request.

npthang committed:
feat(countries): Countries admin page

see #157

npthang committed:
Merge branch 'master' of
https://github.com/php-soft/laravel-marketplace-full-web into
feat/countries/index-page

see #157

npthang committed:
see #157

Commit:
Merge branch 'master' of https://github.com/php-soft/laravel-marketplace-full-web into feat/countries/index-page

npthang committed:
feat(countries): Country admin page

see #157

npthang committed:
feat(countries): Index page

modified:   app/Http/Controllers/Admin/CountryController.php

see #157

npthang committed:
feat(countries): Country admin page

modified:   app/Http/Controllers/Admin/CountryController.php
modified:   resources/views/admin/countries/create.blade.php
modified:   routes/web.php

see #192

npthang committed:
feat(countries): Country admin page

modified:   app/Http/Countrollers/Admin/CountryController.php
modified:   routes/wed.php

see #192

npthang committed:
feat(countries): Country admin page

modified:   app/Http/Controllers/Admin/CountryController.php
modified:   resources/views/admin/countries/index.blade.php
modified:   routes/web.php

see #192

npthang committed:
feat(countries): Country admin page

modified:   app/Http/Controllers/Admin/CountryController.php
new file:   resources/views/admin/countries/edit.blade.php
modified:   resources/views/admin/countries/index.blade.php
modified:   routes/web.php

see #192

huytbt reviewed (commented):
Should code small issue. Not all functions

npthang committed:
feat(countries): Countries admin page

see #192

npthang committed:
feat(countries): Countries admin page

see #192

npthang committed:
feat(countries): Country admin index-page

see #157

npthang committed:
feat(countries): Country admin index-page

modified: routes/wed.php

see #157

npthang committed:
feat(countries): Country admin index-page

modified:   resources/views/layouts/admin.blade.php

see #157

npthang committed:
feat(countries): Country admin index-page

new file:   database/migrations/2017_05_09_111552_create_categories_references_table.php

see #157

huytbt reviewed (approved):
ok
